enterprise/backend/test/metabase_enterprise/sso/integrations/saml_test.clj

```diff
@@ -472,7 +472,7 @@
                         :model_id (:id new-user)
                         :topic    :user-invited
                         :user_id  nil}
-                       (audit-log-test/event :user-invited (:id new-user))))))
+                       (audit-log-test/latest-event :user-invited (:id new-user))))))
```

Comment:
Renamed this function because it was shadowing another `event` variable in a test (I forget where now)